src/user/domain/student.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class Student {
+  @ApiProperty({ type: Number })
+  id: number;
+
+  @ApiProperty({ type: String })
+  name: string;
+
+  @ApiProperty({ type: String })
+  contactEmail: string;
+
+  @ApiProperty({ type: String })
+  description: string;
+
+  @ApiProperty({ type: String })
+  enrollment: string;
+
+  @ApiProperty({ type: String })
+  whatsapp?: string;
+
+  @ApiProperty({ type: String })
+  linkedin?: string;
+
+  @ApiProperty({ type: Number })
+  courseId?: number;
```

Comment:
Adiciona um atributo pra adicionar os dados do curso opcionalmente:

```suggestion
  @ApiProperty({ type: Number })
  courseId?: number;
  
```suggestion
  @ApiProperty({ type: Course })
  course?: Course;
```